docs/topics/mta-web-assessing-apps.adoc

```diff
@@ -22,18 +22,33 @@ The procedure described below uses a built-in questionnaire for assessing the ri
 +
 [NOTE]
 ====
-Only one application can be assessed at a time.
+You can assess only one application at a time.
 ====
-. Click  *Assess*.
+. In the {ProductShortName} user interface, select the *Migration* working mode.
```

Comment:
This step is number 3 in the documentation, [but in the image](https://github.com/windup/windup-documentation/blob/d4f525219a0877f4c64d7d3fa9e8c5124d0d7b48/docs/topics/images/mta-web-apps-details.png), the number is 1.